pkg/kubernetes/types/nodes.go

```diff
@@ -1,10 +1,16 @@
 package types
 
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
 // DebugNodeParams is a type that contains the name, image and command of a node.
 type DebugNodeParams struct {
-	Name    string   `json:"name"`
-	Image   string   `json:"image"`
-	Command []string `json:"command"`
+	Name         string               `json:"name"`
+	Image        string               `json:"image"`
+	Command      []string             `json:"command"`
+	Volumes      []corev1.Volume      `json:"volumes,omitempty"`
+	VolumeMounts []corev1.VolumeMount `json:"volume_mounts,omitempty"`
```

Comment:
Adding all the fields from Volume and VolumnMount is not required as we are only going to run host-networked pods for debug node.
For now lets use 2 fields: `HostPath` and `MountPath`. The validation can be that both should be empty or both should be set. When both are empty, `HostPath` will default to `/` and `MountPath` will default to `/host`.